request-forms/README.md

```diff
@@ -0,0 +1,144 @@
+
+## Using the Forms API
+
+### Create your moffin account
+First you will need an account in the Moffin Sandbox
+environment. You can create one easily in this
+[page](https://moffin.mx/sign_up).
+
+### Generate an Access Token
+Go to the configuration panel in moffin to create an
+*Access Token*.
+
+Go to "Configuración" > "API" > "Crear nuevo token":
+* [Configure token in sandbox](https://sandbox.moffin.mx/configuracion/api)
+* [Configure token in production](https://app.moffin.mx/configuracion/api)
+
+Then you can configure the token as is show in this example.
+```js
+const client = Axios.create({
+  baseURL: 'https://sandbox.moffin.mx/api/v1',
+  headers: {
+  'Authorization': 'Token ' + process.env.API_TOKEN
+  }
+})
+```
+
+You can find more information of which information you
+can send to the API in our [docs](https://moffin.mx/docs).
+
+
+### Form's API flow
+Forms are created by using the `create` endpoint, you should call this function after the user fills or otherwise provides all the required information. Once a Form is created, you need to verify the applicant's contact information, you can simply call `send-sms` and `send-email` to send the verification code to their phone and email respectively. 
+
+Once the user has received their code, you can use the `validate` endpoint to check that it is indeed valid and finally use the `submit` endpoint to complete the form and submit it. 
+
+You can also use the `update` endpoint before submission to update a form's data.
+
+Submission should always be the last step once the form has been created and verified.
+
+### Create a form
+Create a new form using the provided data, keep in mind this doesn't actually submit the form or verify its data. Keep track of the Form ID for the rest of the endpoint calls.
+
+You'll need a Form Config ID, which establishes the format used for the form and some of its configuration. You'll find the Form Config ID for each form defined in your account, but for the following examples and testing in our sandbox environment, you can just use the id `1`.
+
+Please contact support if you need help finding an appropiate Form Config ID for your forms.
+
+```js
+  const { data: form } = await client.post('/form/1', {
+    accountType: 'PF',
+    loanAmount: '10000',
+    email: 'test-dev@moffin.mx',
+    firstName: 'Prueba',
+    phone: '+52 <your phone number>', // Use your own phone number if you wish to test the send-sms endpoint later
+    rfc: 'PRPU800101111',
+    curp: 'PETJ970221HJCRRN09',
+    birthdate: '1997-01-01',
+    address: 'Jaime Balmes',
+    city: 'Cd de Mexico',
+    state: 'CMX',
+    zipCode: '11510',
+    exteriorNumber: '8',
+    neighborhood: 'Los morales',
+    country: 'MX',
+    nationality: 'MX'
+  })
+  console.log('Created Form ID', form.formId)
+```
+
+If the Form is created without issues, you'll get the form ID back, we will use that ID for the remaining requests.
```

Comment:
I would add an example:
```
```suggestion
Once the Form is successfully create you'll get the form ID (`form.id`) back, we will use that ID for the remaining requests.
```

Reply:
Te refieres a un ejemplo del response? Tecnicamente esta justo en las lineas de arriba

Reply:
Igual añadi un snippet del response

Reply:
Si, esa es la idea. Entonces todo bien